data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -115,12 +109,12 @@ podman run -d --net host --name coreos-downloader \
      --restart on-failure \
      --env CACHEURL=${CACHEURL} \
      --env IP_OPTIONS=${IP_OPTIONS} \
-     -v $IRONIC_SHARED_VOLUME:/shared:z ${COREOS_DOWNLOADER_IMAGE} /usr/local/bin/get-resource.sh $RHCOS_BOOT_IMAGE_URL
+     -v $IRONIC_SHARED_VOLUME:/shared:z ${COREOS_DOWNLOADER_IMAGE} /usr/local/bin/get-live-images.sh $RHCOS_BOOT_IMAGE_URL
 
-podman run -d --net host --name ipa-downloader \
-     --restart on-failure \
-     --env CACHEURL=${CACHEURL} \
-     -v $IRONIC_SHARED_VOLUME:/shared:z ${IPA_DOWNLOADER_IMAGE} /usr/local/bin/get-resource.sh
+#podman run -d --net host --name ipa-downloader \
+     #--restart on-failure \
+     #--env CACHEURL=${CACHEURL} \
+     #-v $IRONIC_SHARED_VOLUME:/shared:z ${IPA_DOWNLOADER_IMAGE} /usr/local/bin/get-resource.sh
```

Comment:
I think we're agreed that all the ipa-downloader references can be removed here?

Reply:
Yeah, we don't need these any more.

Reply:
Sure. I'll remove it. I was testing it with it commented out.